main.py

```diff
@@ -0,0 +1,162 @@
+
+RED_SPACESHIP_IMAGE = pygame.image.load(os.path.join('Assets', 'spaceship_red.png'))
+RED_SPACESHIP = pygame.transform.rotate(pygame.transform.scale(RED_SPACESHIP_IMAGE, (SPACESHIP_WIDTH, SPACESHIP_HEIGHT)), 270)
+
+SPACE = pygame.transform.scale(pygame.image.load(os.path.join('Assets', 'space.png')), (WIDTH, HEIGHT))
+
+def draw_window(red, yellow, red_bullets, yellow_bullets, red_health, yellow_health):
+    WIN.blit(SPACE, (0, 0))
+    pygame.draw.rect(WIN, BLACK, BORDER)
+
+    red_health_text = HEALTH_FONT.render("Health: " + str(red_health), 1, WHITE)
+    yellow_health_text = HEALTH_FONT.render("Health: " + str(yellow_health), 1, WHITE)
+    WIN.blit(red_health_text, (WIDTH - red_health_text.get_width() - 10, 10))
+    WIN.blit(yellow_health_text, (10, 10))
+
+    WIN.blit(YELLOW_SPACESHIP, (yellow.x, yellow.y))
+    WIN.blit(RED_SPACESHIP, (red.x, red.y))
+
+    for bullet in red_bullets:
+        pygame.draw.rect(WIN, RED, bullet)
+
+    for bullet in yellow_bullets:
+        pygame.draw.rect(WIN, YELLOW, bullet)
+
+    pygame.display.update()
+
+def yellow_handle_movement(keys_pressed, yellow):
+    if keys_pressed[pygame.K_a] and yellow.x - VEL > 0: #Left
+        yellow.x -= VEL
+    if keys_pressed[pygame.K_d] and yellow.x + VEL + yellow.width < BORDER.x: #Right
+        yellow.x += VEL 
+    if keys_pressed[pygame.K_w] and yellow.y - VEL > 0: #Up 
+        yellow.y -= VEL
+    if keys_pressed[pygame.K_s] and yellow.y + VEL + yellow.height < HEIGHT - 10: #Down
+        yellow.y += VEL
+
+def red_handle_movement(keys_pressed, red):
+    if keys_pressed[pygame.K_LEFT] and red.x - VEL > BORDER.x + BORDER.width: #Left
+        red.x -= VEL
+    if keys_pressed[pygame.K_RIGHT] and red.x + VEL + red.width < WIDTH: #Right
+        red.x += VEL 
+    if keys_pressed[pygame.K_UP] and red.y - VEL > 0: #Up 
+        red.y -= VEL
+    if keys_pressed[pygame.K_DOWN] and red.y + VEL + red.height < HEIGHT - 10: #Down
+        red.y += VEL 
+
+def handle_bullets(yellow_bullets, red_bullets, yellow, red):
+    for bullet in yellow_bullets:
+        bullet.x += BULLET_VEL
+        if red.colliderect(bullet):
+            pygame.event.post(pygame.event.Event(RED_HIT))
+            yellow_bullets.remove(bullet)
+        elif bullet.x > WIDTH:
+            yellow_bullets.remove(bullet)
+
+    for bullet in red_bullets:
+        bullet.x -= BULLET_VEL
+        if yellow.colliderect(bullet):
+            pygame.event.post(pygame.event.Event(YELLOW_HIT))
+            red_bullets.remove(bullet)
+        elif bullet.x < 0:
+            red_bullets.remove(bullet)
+
+def draw_winner(text):
+    draw_text = WINNER_FONT.render(text, 1, WHITE)
+    WIN.blit(draw_text, (WIDTH/2 - draw_text.get_width()/2, HEIGHT/2 - draw_text.get_height()/2)) # Center Winner Name
+    pygame.display.update()
+    pygame.time.delay(5000) #In milliseconds 5000 = 5 seconds
+
+def main():
```

Comment:
beautiful main! I would say adding some context comments would be huge for people new to pygame to learn your code.